cloud/pkg/taskmanager/imageprepullcontroller/image_prepull_controller.go

```diff
@@ -26,6 +26,7 @@ import (
 	"time"
 
 	jsonpatch "github.com/evanphx/json-patch"
+	v1 "k8s.io/api/core/v1"
```

Comment:
"k8s.io/api/core/v1" is enough. or  `corev1 "k8s.io/api/core/v1"`

Reply:
Done